screenpy_selenium/abilities/browse_the_web.py

```diff
@@ -67,18 +68,23 @@ def using_ios(cls) -> Self:
                 capabilities. Default is "iPhone Simulator"
         """
         hub_url = os.getenv("APPIUM_HUB_URL", DEFAULT_APPIUM_HUB_URL)
-        IOS_CAPABILITIES = {
-            "platformName": "iOS",
-            "platformVersion": os.getenv("IOS_DEVICE_VERSION"),
-            "deviceName": os.getenv("IOS_DEVICE_NAME", "iPhone Simulator"),
-            "automationName": "xcuitest",
-            "browserName": "Safari",
-        }
+
+        opts = ArgOptions()
+        opts.set_capability("platformName", "iOS")
+        opts.set_capability("platformVersion", os.getenv("IOS_DEVICE_VERSION"))
+        opts.set_capability(
+            "deviceName", os.getenv("IOS_DEVICE_NAME", "iPhone Simulator")
+        )
+        opts.set_capability("automationName", "xcuitest")
+        opts.set_capability("browserName", "Safari")
+
+        IOS_CAPABILITIES = opts.to_capabilities()
+
```

Comment:
I'm more than a little surprised mypy didn't complain about this sooner.  For sure selenium 4.18 would have triggered it. Perhaps it's just been that long since we updated?

Reply:
Perhaps! Either way, thanks for updating it.